sample/app/package-lock.json

```diff
@@ -7049,6 +7049,19 @@
       "resolved": "https://registry.npmjs.org/hex-color-regex/-/hex-color-regex-1.1.0.tgz",
       "integrity": "sha512-l9sfDFsuqtOqKDsQdqrMRk0U85RZc0RtOR9yPI7mRVOa4FsR/BVnZ0shmQRM96Ji99kYZP/7hn1cedc1+ApsTQ=="
     },
+    "history": {
```

Comment:
Is it intentional that this has both a yarn lockfile and npm lockfile?

Reply:
Yeah. I'm not sure if it's a good idea or not but I thought it was necessary to offer both because we don't know if people prefer yarn or npm. Once we migrate over to npm completely, I can get rid of the yarn flag in the sample app command and the lock file as well.